firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt

```diff
@@ -27,11 +30,56 @@ expect class FirebaseFirestore {
     suspend fun clearPersistence()
     suspend fun <T> runTransaction(func: suspend Transaction.() -> T): T
     fun useEmulator(host: String, port: Int)
-    fun setSettings(persistenceEnabled: Boolean? = null, sslEnabled: Boolean? = null, host: String? = null, cacheSizeBytes: Long? = null)
     suspend fun disableNetwork()
     suspend fun enableNetwork()
 }
 
+val FirestoreSettings.Companion.CACHE_SIZE_UNLIMITED get() = -1L
+val FirestoreSettings.Companion.DEFAULT_HOST get() = "firestore.googleapis.com"
+
+expect class FirestoreSettings {
+
+    companion object {}
+
+    interface Builder {
```

Comment:
make this a class with the constructors available here to match android:
https://firebase.google.com/docs/reference/android/com/google/firebase/firestore/FirebaseFirestoreSettings.Builder